### ISSUE TITLE
Can I choose several emojis to form a group? And configure its navigation and header？

### ISSUE THREAD

**Issue by Cribug8080** [ealush/emoji-picker-react#379]

**Comment by zehawki** [comment]
I came here looking for the same, and see its been raised a few times earlier as well:

1. https://github.com/ealush/emoji-picker-react/issues/216
2. https://github.com/ealush/emoji-picker-react/issues/350

Basically - it would be great if its possible to create a custom group from a set of emojis (not custom), and open into that on 1st load

**Comment by zehawki** [comment]
See this https://github.com/nolanlawson/emoji-picker-element/issues/323

**Comment by ealush** [comment]
Sorry for responding late. I have been traveling the past several weeks.

That's an interesting proposal. So if I understand the ask correctly, we're talking about a group that's comprised of just _any_ emoji, not necessarily belonging to the same group, right?

Follow up questions:
1. What would be the icon for that category? Is part of the proposal adding custom group icons?
2. Will emojis be able to appear more than once, meaning - once in the "favorites group" and once in their original group?

**Comment by Cribug8080** [comment]
@ealush 
Thanks for the reply.

First, I wish I could customize the icon, but this is not a core point.
Second, it's true that the emoji appears multiple times

This is a great program and could have this third option, "SUGGESTED" meet my demand

**Comment by zehawki** [comment]
> So if I understand the ask correctly, we're talking about a group that's comprised of just any emoji, not necessarily belonging to the same group, right?

Yup, thats right. Its a selection from the emoji master list, based on whatever requirement the dev has in mind.

> What would be the icon for that category? Is part of the proposal adding custom group icons?

Allowing this to be customizable would be quite perfect.

> Will emojis be able to appear more than once, meaning - once in the "favorites group" and once in their original group?

Yes! Placement into the fav (aka custom) group is only to help the end user with a quick selection, and doesnt affect the original groups / tabs / functionality etc.

> Currently the suggested category has: either "frequent" or "recent", the third option suggested would be a predefined list defined by you.

Yes, thats exactly how I envisaged it, which is why I was pointing to https://github.com/nolanlawson/emoji-picker-element/issues/323, where its implemented as a bar, and that limits the number of emoji, whereas in case of this package, it can be a custom group and hence many (say 12) emoji can be displayed for quick selection.

**Comment by ealush** [comment]
Shipped in #532 (merged as ad4cc5e8, released in 4.21.1): group customs via `group` and list one `{ category: Categories.CUSTOM, group, name }` entry per group in `categories` — each group gets its own navigation tab and header, ordered by the array. See CUSTOMIZATION.md (Grouping custom emojis) and the CustomEmojisGrouped story.

**Comment by ealush** [comment]
Correction to my closing comment above: PR #532 is merged (ad4cc5e8), but I incorrectly stated it was released in 4.21.1 — that release contains only the #531 fix. The custom-groups feature is unreleased as of now and will go out in the next minor release (4.22.0) via semantic-release. Sorry for the confusion.

**Comment by ealush** [comment]
Reopening with a correction: my earlier closing comment overstated the coverage. PR #532 groups image-based *custom* emojis into named sections — it does not support composing a group from built-in Unicode emojis while retaining them in their original categories, which is what this issue asks for. That capability remains unsupported. Keeping this open to track it.
